code/modules/clothing/spacesuits/rig/rig_construction.dm

```diff
@@ -0,0 +1,428 @@
+	icon_base = "hacker"
+	icon_state = "hacker1"
+	rig_type = /obj/item/weapon/rig/light/hacker
+	board_type = /obj/item/weapon/circuitboard/rig_assembly/illegal/hacker
+	target_board_type = /obj/item/weapon/circuitboard/rig_assembly/illegal/targeting/hacker
+
+/obj/item/rig_assembly/combat/illegal/stealth
+	name = "stealth suit control module assembly"
+	desc = "An assembly for a highly advanced and expensive suit designed for covert operations."
+	icon_base = "stealth"
+	icon_state = "stealth1"
+	rig_type = /obj/item/weapon/rig/light/stealth
+	board_type = /obj/item/weapon/circuitboard/rig_assembly/illegal/stealth
+	target_board_type = /obj/item/weapon/circuitboard/rig_assembly/illegal/targeting/stealth
+	origin_tech = list(TECH_MATERIAL = 6, TECH_ENGINEERING = 5, TECH_MAGNET = 4, TECH_POWER = 5, TECH_COMBAT = 6, TECH_ILLEGAL = 6)
+
+/datum/construction/reversible/rig_assembly
+	result = null
+	var/obj/item/weapon/circuitboard/rig_assembly/board_type = null
+	var/obj/item/weapon/circuitboard/rig_assembly/target_board_type = null
+
+/datum/construction/reversible/rig_assembly/custom_action(index as num, diff as num, atom/used_atom, mob/user as mob)
+	var/obj/item/I = used_atom
+	if(I.iswelder())
+		var/obj/item/weapon/weldingtool/W = I
+		if (W.remove_fuel(0, user))
+			playsound(holder, 'sound/items/Welder2.ogg', 50, 1)
+		else
+			return 0
+	else if(I.iswrench())
+		playsound(holder, 'sound/items/Ratchet.ogg', 50, 1)
+
+	else if(I.isscrewdriver())
+		playsound(holder, 'sound/items/Screwdriver.ogg', 50, 1)
+
+	else if(I.iswirecutter())
+		playsound(holder, 'sound/items/Wirecutter.ogg', 50, 1)
+
+	else if(I.iscoil())
+		var/obj/item/stack/cable_coil/C = used_atom
+		if(C.use(4))
+			playsound(holder, 'sound/items/Deconstruct.ogg', 50, 1)
+		else
+			to_chat(user, "There's not enough cable to finish the task.")
+			return 0
+	else if(istype(I, /obj/item/stack))
+		var/obj/item/stack/S = I
+		if(S.get_amount() < 5)
+			to_chat(user, "There's not enough material in this stack.")
+			return 0
+		else
+			S.use(5)
+	return 1
+
+/datum/construction/reversible/rig_assembly/civilian
+	steps = list(
+				//1
+				list("key"=/obj/item/weapon/weldingtool,
+					"backkey"=/obj/item/weapon/wrench,
+					"desc"="External armor is wrenched"),
+				//2
+				list("key"=/obj/item/weapon/wrench,
+						"backkey"=/obj/item/weapon/crowbar,
+						"desc"="External armor is installed"),
+				//3
+				list("key"=/obj/item/stack/material/steel,
+						"backkey"=/obj/item/weapon/screwdriver,
+						"desc"="Central control module is secured"),
+				//4
+				list("key"=/obj/item/weapon/screwdriver,
+						"backkey"=/obj/item/weapon/crowbar,
+						"desc"="Central control module is installed"),
+				//5
+				list("key"=null,
+						"backkey"=/obj/item/weapon/wirecutters,
+						"desc"="The wiring is adjusted"),
+				//6
+				list("key"=/obj/item/weapon/wirecutters,
+						"backkey"=/obj/item/weapon/screwdriver,
+						"desc"="The wiring is added"),
```

Comment:
with a screwdriver?

Reply:
Yes, that is to go back to step of removing wire. Unfortunately I cannot make it wirecutters as to proceed to the next step you also need wirecutters.